src/modules/list.marko

```diff
@@ -0,0 +1,359 @@
+<div id="list">
+    <section-header metadata=input.metadata name=input.name/>
+    <p>A list component is used to display certain types of data in a structured format. It can be used to display a list of items, a list of links, or a list of buttons.</p>
+    <p>This can be placed inside a container and will resize to fit the contents. The list expands to a maxumum of 480px.</p>
+
+    <h3 id="list-default">Default List</h3>
+   <div class="demo">
+        <div class="demo__inner">
+            <div class="list">
```

Comment:
Is this `div` necessary? Can we just have `<ul class="list">`?

Reply:
Yea, I second this. It looks like the only thing applied to this is `max-width: 480px;`, which can easily be applied to the `ul`.

Reply:
This is actually needed based on what i spoke with Ian.
So the idea is that we could put in other elements inside list such a s fieldset (for checkboxes or radio). We dont necessarily want the same contents always. If we put in other content and it would style it the same way. 
This hasn't been done yet and will be done as a phase 2.